doc/api/api_changes.rst

```diff
@@ -362,6 +366,23 @@ The `matplotlib.lines` module no longer imports the `pts_to_prestep`,
 `pts_to_midstep` and `pts_to_poststep` functions from the `matplotlib.cbook`
 module.
 
+Invalid (Non-finite) Axis Limit Error
+-------------------------------------
+
+When using :func:`~matplotlib.axes.Axes.set_xlim` and
+:func:`~matplotlib.axes.Axes.set_ylim`, passing non-finite values now
+results in a ``ValueError``. The previous behavior resulted in the
+limits being erroneously reset to ``(-0.001, 0.001)``.
+
+`scatter` and `Collection` offsets are no longer implicitly flattened
```

Comment:
This doesn't actually link; not sure if you want it to; since links are on the next line, maybe just double-backticks.

Reply:
I guess I needed to use `~.scatter` but yes, double bq are fine

Reply:
You don't want to put links in section headings, it conflicts with how sphinx links to/from the section headings.